specification/applicationinsights/resource-manager/Microsoft.Insights/preview/2020-03-01-preview/componentLinkedStorageAccounts_API.json

```diff
@@ -0,0 +1,289 @@
+            "$ref": "./examples/ComponentLinkedStorageAccountsUpdate.json"
+          }
+        }
+      },
+      "delete": {
+        "description": "Delete linked storage accounts for an Application Insights component.",
+        "operationId": "ComponentLinkedStorageAccounts_Delete",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceNameParameter"
+          },
+          {
+            "$ref": "#/parameters/StorageTypeParameter"
+          }
+        ],
+        "responses": {
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          },
+          "200": {
+            "description": "Successful request when deleting linked storage accounts for an Application Insights component."
+          },
+          "204": {
+            "description": "Successful request when deleting linked storage accounts for an Application Insights component."
+          }
+        },
+        "x-ms-examples": {
+          "ComponentLinkedStorageAccountsDelete": {
+            "$ref": "./examples/ComponentLinkedStorageAccountsDelete.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "ErrorResponse": {
+      "description": "Describe the format of an Error response.",
+      "type": "object",
+      "properties": {
+        "code": {
+          "description": "Error code",
+          "type": "string"
+        },
+        "message": {
+          "description": "Error message indicating why the operation failed.",
+          "type": "string"
+        }
+      }
+    },
+    "ComponentLinkedStorageAccounts": {
+      "description": "An Application Insights component linked storage accounts",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v1/types.json#/definitions/Resource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/LinkedStorageAccountsProperties",
+          "description": "The properties of the linked storage accounts.",
+          "x-ms-client-flatten": true
+        }
+      }
+    },
+    "ComponentLinkedStorageAccountsPatch": {
+      "description": "An Application Insights component linked storage accounts patch",
+      "type": "object",
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/LinkedStorageAccountsProperties",
```

Comment:
>LinkedStorageAccountsProperties [](start = 33, length = 31)

 are you sure linkedStorageAccount property is patchable ?  (can user update after creating the resource)

Reply:
Yes, by design